metrics/api/src/main/java/io/helidon/metrics/api/MetricsSettings.java

```diff
@@ -33,12 +33,12 @@
 public interface MetricsSettings {
 
     /**
-     * Returns default metrics settings based on default config.
+     * Returns default metrics settings based on the {@code metrics} section of the default config.
      *
-     * @return new settings reflecting the default config
+     * @return new settings reflecting the default metrics config
      */
     static MetricsSettings create() {
-        return create(Config.create());
```

Comment:
How incompatible is this change? I'm concerned that users will have defined a top-level config key of "tags" to get metrics global tags to work -- or maybe that is unlikely.